api-tests/core/content-manager/api/number-of-draft-relations.test.api.js

```diff
@@ -323,46 +322,73 @@ describe('CM API - Basic', () => {
       url: `/content-manager/collection-types/api::product.product/${product.id}/actions/countDraftRelations`,
     });
 
-    expect(body.data).toBe(8);
+    expect(body.data).toBe(12);
   });
 
-  test('Return 12 when there are 12 drafts (1 xToOne & 2 xToMany on ct, compo, comporep, dz)', async () => {
-    const categoryId = data.categories[0].id;
-    const draft1Id = data.categoriesdp.draft[0].id;
-    const draft2Id = data.categoriesdp.draft[1].id;
+  test('Return 8 when there are 8 drafts in a non default locale', async () => {
+    const localeQuery = `plugins[i18n][locale]=${locale}`;
```

Comment:
Reminder for me that this might change after pr https://github.com/strapi/strapi/pull/18975